opennms-services/src/main/java/org/opennms/netmgt/translator/EventTranslator.java

```diff
@@ -156,15 +156,15 @@ public void onEvent(Event e) {
             return;
         }
 
-        if (!m_config.isTranslationEvent(e)) {
-            LOG.debug("onEvent: received event that matches no translations: \n", EventUtils.toString(e));
-            return;
-        }
-
-        LOG.debug("onEvent: received valid registered translation event: \n", EventUtils.toString(e));
-
         List<Event> translated = m_config.translateEvent(e);
         if (translated != null) {
+            if (translated.isEmpty()) {
+                LOG.debug("onEvent: received event that matches no translations: \n", EventUtils.toString(e));
```

Comment:
There's no `{}` placeholder to show the event in this message. We should also avoid calling `EventUtils.toString()` unless DEBUG logging is enabled.

(I realize this was existing code, but we may as well change it while we're here.)